pkg/action/archive_test.go

```diff
@@ -221,16 +221,23 @@ func TestScanArchive(t *testing.T) {
 		t.Fatalf("render: %v", err)
 	}
 
-	bc := malcontent.Config{
+	rfs := []fs.FS{rules.FS, thirdparty.FS}
+	yrs, err := CachedRules(ctx, rfs)
+	if err != nil {
+		t.Fatalf("rules: %v", err)
+	}
+
+	mc := malcontent.Config{
 		Concurrency: runtime.NumCPU(),
 		IgnoreSelf:  false,
 		MinFileRisk: 0,
 		MinRisk:     0,
 		Renderer:    r,
-		RuleFS:      []fs.FS{rules.FS, thirdparty.FS},
+		RuleFS:      rfs,
```

Comment:
Why do we have to pass both Rules and RuleFS in?

Reply:
Updated that in [`1531cce` (#647)](https://github.com/chainguard-dev/malcontent/pull/647/commits/1531cce2dbfa64be7803ec382ee6925ecb17cecb).